<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SobelDialog</class>
 <widget class="QDialog" name="SobelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>201</width>
     <height>250</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Operation</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="SobelComboBox">
      <item>
       <property name="text">
        <string>select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sobel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laplacian</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="sobelApplyBlurCheckBox">
      <property name="text">
       <string>Apply Blur</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Kernel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="SobelKernelspinBox">
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>dx</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="SobelDxSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>dx weight</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="SobelDxWeightSpinBox">
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>0.900000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>dy</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="SobelDySpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Delta</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="SobelScaleSpinBox">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="SobelDeltaSpinBox">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
